chroma-agent/chroma_agent/agent_client.py

```diff
@@ -296,7 +296,10 @@ def terminate(self, plugin_name):
         else:
             daemon_log.info("SessionTable.terminate %s/%s" % (plugin_name, session.id))
             session.teardown()
-            del self._sessions[plugin_name]
+            try:
+                del self._sessions[plugin_name]
+            except KeyError:
+                pass
```

Comment:
we could do, didn't feel it was necessary at the time. want me to add some logging?